libraries/provider_runit_service.rb

```diff
@@ -224,7 +224,7 @@ def whyrun_supported?
           end
 
           # lsb_init
-          if node['platform'] == 'debian'
+          if node['platform'] == 'debian' || node['platform'] == 'ubuntu'
```

Comment:
if platform_family?('debian') would work better here by also roping in any derivative distros.

Reply:
ack. Would chef still find templates/debian/init.d.erb (source 'init.d.erb' used below) for these derivative platforms? I'm leaning for a - no, thus possibly move it into template/default/ ?